Extension/src/LanguageServer/ui.ts

```diff
@@ -36,10 +36,18 @@ interface ConfigurationStatus {
 }
 
 export class UI {
-    private configStatusBarItem: vscode.StatusBarItem;
-    private browseEngineStatusBarItem: vscode.StatusBarItem;
-    private intelliSenseStatusBarItem: vscode.StatusBarItem;
-    private referencesStatusBarItem: vscode.StatusBarItem;
+    private configStatusBarItem!: vscode.LanguageStatusItem;
+    private browseEngineStatusBarItem!: vscode.LanguageStatusItem | undefined;
```

Comment:
It seems like the `| undefined` could be replaced by `browseEngineStatusBarItem?:` but I'm not sure what the `!:` is as mentioned earlier.

Reply:
Yeah you're right. It was from a previous idea that didn't work out